demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+---
+title: 'Quickstart: Deploy an AKS cluster by using Azure CLI'
+description: Learn how to quickly create a Kubernetes cluster, deploy an application, and monitor performance in Azure Kubernetes Service (AKS) using the Azure CLI.
+services: container-service
+ms.topic: quickstart
+ms.date: 11/01/2022
+ms.custom: H1Hack27Feb2017, mvc, devcenter, seo-javascript-september2019, seo-javascript-october2019, seo-python-october2019, devx-track-azurecli, contperf-fy21q1, mode-api
+#Customer intent: As a developer or cluster operator, I want to quickly create an AKS cluster and deploy an application so that I can see how to run and monitor applications using the managed Kubernetes service in Azure.
+---
+
+# Quickstart: Deploy an Azure Kubernetes Service cluster using the Azure CLI
+
+Azure Kubernetes Service (AKS) is a managed Kubernetes service that lets you quickly deploy and manage clusters. In this quickstart, you will:
+
+* Deploy an AKS cluster using the Azure CLI.
+* Run a sample multi-container application with a web front-end and a Redis instance in the cluster.
+
+:::image type="content" source="media/quick-kubernetes-deploy-portal/azure-voting-application.png" alt-text="Screenshot of browsing to Azure Vote sample application.":::
+
+This quickstart assumes a basic understanding of Kubernetes concepts. For more information, see [Kubernetes core concepts for Azure Kubernetes Service (AKS)][kubernetes-concepts].
+
+[!INCLUDE [quickstarts-free-trial-note](../../../includes/quickstarts-free-trial-note.md)]
+
+To learn more about creating a Windows Server node pool, see [Create an AKS cluster that supports Windows Server containers](quick-windows-container-deploy-cli.md).
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment.md)]
+
+- This article requires version 2.0.64 or later of the Azure CLI. If using Azure Cloud Shell, the latest version is already installed.
+
+- The identity you are using to create your cluster has the appropriate minimum permissions. For more details on access and identity for AKS, see [Access and identity options for Azure Kubernetes Service (AKS)][aks-identity-concepts].
+
+- If you have multiple Azure subscriptions, select the appropriate subscription ID in which the resources should be billed using the
+[az account][az-account] command.
+
+- Verify *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* providers are registered on your subscription. These are Azure resource providers required to support [Container insights][azure-monitor-containers]. To check the registration status, run the following commands:
+
+    ```azurecli
+    az provider show -n Microsoft.OperationsManagement -o table
+    az provider show -n Microsoft.OperationalInsights -o table
+    ```
```

Comment:
I do not test/verify success of every command. I agree that a north star should be to test every single command. However, given time I am primarily only testing the end result. I am making the assumption that if we get to the end of the tutorial and an AKS cluster is deployed with a working website that this step also succeeded.

Reply:
I feel this approach is suboptimal. To show the true value of this testing framework it is not sufficient to say "it failed". We need to tell the docs team, who are responsible for all docs and SMEs in none, why it failed and where. This will often allow them to fix it without engaging a SME. If they do need to engage the SME they will be able to provide the error output from Innovation Engine which, more often than not, will be enough for the SME to identify the fix. Thus making the process significantly more efficient.

Furthermore, results sections make the content more easy to follow for human readers as they can proceed with confidence that the output they got was correct.